test/e2e/pages/vl-multi-select.page.js

```diff
@@ -1,13 +1,92 @@
 const VlMultiSelect = require('../components/vl-multi-select');
+//const { VlDatepicker } = require('vl-ui-datepicker').Test;
```

Comment:
In comment, niet meer nodig?

Reply:
Mag uit comment vanaf het punt dat de datepicker stable is en de datepicker testen terug aangezet kunnen worden.